app/views/coronavirus_landing_page/_accordion_sections.html.erb

```diff
@@ -0,0 +1,14 @@
+  <% accordion_contents = [] %>
+  <% t("coronavirus_landing_page.sections").each do | section | %>
+    <% contents = capture do %>
+      <%= render( partial: 'section', locals: section ) %>
+    <% end %>
+    <%
+      accordion_contents << {
+        title: section[:title],
+        id: section[:title].underscore,
+        panel: contents
+      }
+    %>
+  <% end %>
+  <%= render 'components/accordion', sections: accordion_contents %>
```

Comment:
You probably want the accordion component from the gem?

Reply:
Yeah, I wasn't sure, given that it says "experimental".  I used the same thing as the world location news pages do.

Reply:
I've pushed that as a new commit